test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/IncludeSearchTests.cs

```diff
@@ -10,6 +10,7 @@
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using System.Web;
+using DotLiquid.Tags;
```

Comment:
🤔

Reply:
Ug, that is probably from an autocomplete when I was adding the word tag to the search query.